rke/templates/templates.go

```diff
@@ -70,6 +70,7 @@ const (
 	*/
 	aciv500 = "aci-v5.0.0"
 	aciv523 = "aci-v5.2.3"
+	aciv5233 = "aci-v5.2.3.3"
```

Comment:
Nit, but we want variables and values to align, see other blocks like this.

Reply:
addressed